website/docs/admin_colorspace.md

```diff
@@ -0,0 +1,193 @@
+---
+id: admin_colorspace
+title: Colorspace settings
+sidebar_label: Colorspace settings
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+:::important Actual limitations
+
+This is a work in progress. The current implementation is a bit limited and will be improved in the future.
+Currently only limited amount of hosts are supported and only publishing is implemented. Loading is not yet supported.
+
+:::
+
+## Key concept of colorspace distribution
+
+### Host groups
+![hosts](assets/settings/admin_colorspace_1.png)
+
+Hosts are divided into 3 groups regarding the level of control of a DCC's native colorspace manageability.
+
+1. **OCIO managed** - This application's colorspace management can be controlled through OpenPype settings. Specifically, the configured OpenColorIO (OCIO) config path is utilized in the application's workfile. Additionally, the File Rules feature can be leveraged for both publishing and loading procedures.
+
+2. **Remapped internal colorspace** - This application includes internal color management functionality, but it does not offer external control over this feature. To address this limitation, OpenPype uses mapping rules to remap the native colorspace names used in the internal color management system to the OpenColorIO (OCIO) color management system. Remapping feature is used in Publishing and Loading procedures.
+
+3. **Derived colorspace** - This application does not include any built-in color management capabilities, OpenPype offers a solution to this limitation by deriving valid colorspace names for the OpenColorIO (OCIO) color management system from file paths, using File Rules feature (publishing only).
+
+
+### How does hosts overrides work
```

Comment:
> I think it's also good to move this chapter down so that the "global setting" for color management is discussed first and it must be absolutely clear to the user that host or global settings do nothing unless the global color management setting is enabled. Right?

This is in the **key concept** category and configuration section with global and host sub categories follow, so I would keep the order as it is.